lib/kinesis/manager/Manager.rb

```diff
@@ -0,0 +1,212 @@
+require "common/manager/Manager"
+require "conf/Configuration"
+require "kinesis/Kinesis"
+require "kinesis/loader/Loader"
+require "kinesis/models/StreamConfig"
+require "kinesis/models/StreamDiff"
+
+require "aws-sdk"
+
+module Cumulus
+  module Kinesis
+    class Manager < Common::Manager
+
+      def initialize
+        super()
+        @create_asset = true
+        @client = Aws::Kinesis::Client.new(region: Configuration.instance.region, profile: Configuration.instance.profile)
```

Comment:
I have tried to do that before in other modules with mixed results. It's safest just to use a new client wherever you need it, since accessing the static client from the main module seems to be Nil sometimes and not Nil other times, depending on which methods get called first.